CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v1.26.2 (2024-11-04)
+
+## Release Highlights
+* Update kernels: 5.10.227, 5.15.168, 6.1.112-124 ([bottlerocket-core-kit#231](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/231), [bottlerocket-core-kit#235](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/235))
```

Comment:
Since the inception of the core-kit, I don't believe we've listed kernel updates as a Release Highlight. Any specific reason we're including it here?